params/protocol_params.go

```diff
@@ -163,8 +169,22 @@ func (t CodeFormat) String() string {
 }
 
 func (t CodeFormat) Validate() bool {
-	if t < CodeFormatLast {
+	if t&0b01111111 < CodeFormatLast { // IstanbulHF field is irrelevant with CodeFormat.
 		return true
 	}
 	return false
 }
+
+func (t CodeFormat) IsIstanbulHF() bool {
```

Comment:
Minor comment, however, how about considering `AfterInstanbulHF`? 
As "a CodeFormat is Instanbul HF" sounds a bit confusing. How do you think?

Reply:
Thanks! I'll change the name to `AfterIstanbulHF`.

Reply:
Changed the name to `IsDeployedAfterIstanbulHF` (https://github.com/klaytn/klaytn/commit/b62138adad3b89433c763458df317a578be8d84a)